src/main/java/com/epam/dynamos3/services/MapperServiceJacksonImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.dynamos3.services;
+
+import com.epam.dynamos3.pojos.Product;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MapperServiceJacksonImpl implements MapperService{
```

Comment:
JacksonMapperService could be a better name

Reply:
Done! Thxs!